jest.config.js

```diff
@@ -81,6 +81,7 @@ module.exports = {
     "\\.css$": "identity-obj-proxy",
     "react-markdown":
       "<rootDir>/node_modules/react-markdown/react-markdown.min.js",
+    "^@jobber/hooks/(.*)$": "<rootDir>/packages/hooks/dist/$1",
```

Comment:
I wonder if this and the other module mapping we added should be in a single place so it's easy to find.

Like create a config file and import it to the things that needs it.

```js
// moduleMap.config.js

export const hooks = {
  storybook: ...
  jest: ...
  ...
}
```

Reply:
What other module mapping?

Reply:
https://github.com/GetJobber/atlantis/blob/1da63751c24367d216c978ce060b7e30bf78463b/.eslintrc.js#L6
https://github.com/GetJobber/atlantis/blob/1da63751c24367d216c978ce060b7e30bf78463b/.storybook/main.js#L116
https://github.com/GetJobber/atlantis/blob/1da63751c24367d216c978ce060b7e30bf78463b/packages/components/tsconfig.json#L10-L12

Reply:
Can we do it in a follow-up?

Reply:
yup